docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
<p>data =&gt; code</p><p> </p><p>The USGS page doesn't seem to have a list of data repositories, this is just confusion in the sentence structure, reorder to put the link last after explaining it's purpose.</p><p> </p><p>"If you are NASA-Funded: " mixes in Data, which I don't think we are ready to address yet. Nothing else in the Document covers data yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>